Testing/SystemTests/tests/analysis/VesuvioTesting.py

```diff
@@ -0,0 +1,514 @@
+from mantid.simpleapi import *
```

Comment:
I did just forget to remove this from `Framework/PythonInterface` but actually on closer inspection it is  required by both. I agree we don't want a duplicate. Do you know what the best practice would be in terms of referencing the VesuvioTesting.py from both system tests and unit tests? Also where it should go if both are using it?